pkg/debug/ssh.sh

```diff
@@ -7,4 +7,8 @@ echo -1 > /proc/sys/kernel/perf_event_paranoid
 KEYS=$(find /etc/ssh -name 'ssh_host_*_key')
 [ -z "$KEYS" ] && ssh-keygen -A >/dev/null 2>/dev/null
 
+mkdir -p /run/debug/usr/bin
+cp /usr/bin/lshw /run/debug/usr/bin/.
+cp /usr/bin/spec.sh /run/debug/usr/bin/.
```

Comment:
We should either move this into whatever container you need them to be in or come up with some kind of a mounting solution -- this kind of copying shouldn't happen (it is OK for a WIP but not for when we need to merge this).

Reply:
I was thinking if the 'edge-view' is really used, and if we still keep the 'debug' container, then this allows us not to duplicate the two sets of services, if we later remove the 'debug' container, then those two can be moved into 'edge-view' container. but i can drop those to services for now.